ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -358,7 +365,7 @@ static int HandleVnodeOperation(
     
     if (isDeleteAction)
     {
-        if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, &root, &vnodeFsidInode, &kauthResult, kauthError))
+        if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, UserPolicy_AllowAny, &root, &vnodeFsidInode, &kauthResult, kauthError))
```

Comment:
Could you add comments describing why we're using `UserPolicy_AllowAny` here and `UserPolicy_UserInitiatedOnly` below?